test/intl_suggestors/intl_importer_test.dart

```diff
@@ -148,7 +154,7 @@ void main() {
                 isUndefinedIntlError(e) || isFakeUriError(e),
             expectedOutput: /*language=dart*/ '''
               import 'package:over_react/over_react.dart';
-              import 'package:test_project/src/intl/test_project_intl.dart';
+              import '../../../../src/intl/test_project_intl.dart';
```

Comment:
There should be a test case for a package import in the same package. All these tests produce relative imports.